justfile

```
@@ -0,0 +1,14 @@
+_default:
+  @just --list
+
+flavors := "latte frappe macchiato mocha"
+accents := "rosewater flamingo pink mauve red maroon peach yellow green teal sky sapphire blue lavender"
+
+build:
+  @for flavor in {{flavors}}; do \
+    whiskers -f "$flavor" "./fcitx5.tera"; \
+    for accent in {{accents}}; do \
+      cp "./assets/$flavor-radio.png" "./src/catppuccin-$flavor-$accent/radio.png"; \
+      cp "./assets/$flavor-arrow.png" "./src/catppuccin-$flavor-$accent/arrow.png"; \
+    done; \
+  done
```

You are using a matrix for the Whiskers build process, so you don't need to run Whiskers for each flavor in this loop.
```suggestion
  #!/usr/bin/env bash
  whiskers fcitx5.tera
  whiskers --list-flavors -o plain | while read -r flavor; do
    whiskers --list-accents -o plain | while read -r accent; do
      cp "./assets/$flavor-radio.png" "./src/catppuccin-$flavor-$accent/radio.png"
      cp "./assets/$flavor-arrow.png" "./src/catppuccin-$flavor-$accent/arrow.png"
    done
  done
```

---

Thanks, this looks way cleaner